core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -98,6 +100,8 @@
     @Nullable
     private volatile ServerPort primaryActivePort;
 
+    private ServerBootstrap serverBootstrap;
```

Comment:
nit: How about setting it back to `null` when the server stops, so that all channel services are GC'd even if a user forgot to dereference a `Server`?

Reply:
So this is `@Nullabe`?